src/connectors/radiojavan.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+Connector.playerSelector = '.webPlayer';
+
+Connector.artistSelector = () => {
+	var artist = $('.mainPanel .artist').attr('data');
+	return artist;
+};
+
+Connector.trackSelector = () => {
+	var track = $('.mainPanel .song').attr('data');
+	return track;
+};
+
+Connector.artistTrackSelector = () => {
+	var track = $('.mainPanel .song').attr('data');
+	var artist = $('.mainPanel .artist').attr('data');
+	return { artist, track };
+};
+
+Connector.playButtonSelector = '#mp3_play';
```

Comment:
Is this correct? Looking at the DOM It looks like it should be using the `#mp3_play` child span based on the class.

Reply:
```
Connector.isPlaying = () => {
	return $('#mp3_play > span').hasClass('musicPause');
};
```

Reply:
It also could be replaced by:
```js
Connector.pauseButtonSelector = '#mp3_play .musicPause';
// or
Connector.playButtonSelector = '#mp3_play .musicPlay';
```
but the function is acceptable too.

Reply:
@alexesprit I think your approach is clearer to the reader :+1: